scripts/parse_skill_data.py

```diff
@@ -151,7 +151,7 @@ def update_neon_mark_2_html():
 
     all_skills.sort(key=lambda s: s[0].split('>')[1].split('\\<')[0])
     all_skills.insert(0, ["Skill", "Summary", "Examples"])
-    mk2_skills_path = join(dirname(dirname(__file__)), 'html',
+    mk2_skills_path = join(dirname(__file__), 'html',
```

Comment:
This just changes the output file path from `html/mark_2_default_skills.html` to `mark_2_default_skills.html`. I prefer to keep all html in that one subdirectory to help organize things here, but could move it to the repository root if we need to